includes/core-changes/aspnetcore/3.0/sharedfx-app-shared-framework-assemblies.md

```diff
@@ -47,7 +47,7 @@ It won't be necessary for projects to consume assemblies in `Microsoft.AspNetCor
 
 This change doesn't apply to all binaries referenced via `Microsoft.AspNetCore.App` in ASP.NET Core 2.x. Notable exceptions include:
 
-- `Microsoft.Extensions` libraries that continue to target .NET Standard will be available as NuGet packages (see https://github.com/dotnet/extensions).
+- `Microsoft.Extensions` libraries that continue to target .NET Standard will be available as NuGet packages (see <https://github.com/dotnet/extensions>).
```

Comment:
Not related to the changes, but it seems that "will be" should be changed to "are".

<https://www.nuget.org/packages?q=Microsoft.Extensions>

Reply:
Yeah, it seems like the whole document is written in the future. I'll leave that up to a real tech writer to fix separately :)